README.md

```
@@ -1,177 +1,293 @@
+
+# (선택) DB 마이그레이션
+# alembic upgrade head
+
+uvicorn backend.main:app --reload
+# 기본: http://localhost:8000
+
 
 ---
 
-## 6. 룰 & 가이드라인 (Rules & Guidelines)
+### 4.5 AI 워커 실행 (AI Worker)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)